Dz2.py

```diff
@@ -0,0 +1,10 @@
+i = (input("Введите текст: "))
+if (i.isdigit()) == True:

```

Comment:
There is no sense in using parentheses around i.isdigit(), and you can skip the comparison `== True` since in the end `if` will do it for you.